app/views/devise/registrations/_registration_form.html.erb

```diff
@@ -1,60 +1,70 @@
-<div class="new-article-pitch">
-  <% if @new_article_not_logged_in %>
-    <div class="prefill-non-logged-in-message">
-      <div class="registration__logo-container">
-        <img class="secondary-logo" src="<%= cloudinary(safe_logo_url(SiteConfig.secondary_logo_url), 190) %>"
-             alt="Secondary <%= community_name %> logo" />
-      </div>
-      <p>
-        <b>Welcome to <%= community_name %></b>
-      </p>
-      <p>
-        Sign in below to compose your post and share it with the community.
-      </p>
+<section class="crayons-layout">
+  <div class="registration crayons-card">
+    <div class="registration__image-container">
+      <img class="registration__image" src="<%= cloudinary(safe_logo_url(SiteConfig.secondary_logo_url), 190) %>"
+            alt="Secondary <%= community_name %> logo" />
     </div>
-  <% else %>
-    <div class="registration__logo-container">
-      <img class="secondary-logo" src="<%= cloudinary(safe_logo_url(SiteConfig.secondary_logo_url), 190) %>"
-           alt="Secondary <%= community_name %> logo" />
+    <div class="registration__content">
+      <h1 class="registration__title">
+        Welcome to <%= community_name %>
+      </h1>
+      <% if any_selfserve_auth? %>
+        <p class="registration__description">
+community_members_label
+                                                                  %>. Registration is by invitation only. Sign in
+          by social auth if you don't have a password set.
```

Comment:
I'm contemplating breaking down this chunk of paragraph so that it's not overwhelming. Maybe the text `"Sign in by social auth if you don't have a password set."` can go below the OAuth buttons.